app/constants.js

```diff
@@ -2,6 +2,7 @@ import path from 'path';
 import electron from 'electron';
 import { platform, homedir } from 'os';
 import getAppPath from './utils/getAppPath';
+import os from 'os';
```

Comment:
Platform is already imported so it's enough to call platform() instead of os.platform() and remove this line